src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderReminderNotification.java

```diff
@@ -49,69 +50,91 @@ public class AwaitingConditionalOrderReminderNotification implements ApplicantNo
 
     @Override
     public void sendToApplicant1(final CaseData caseData, final Long id) {
-        log.info("Sending reminder to applicant 1 that they can apply for a conditional order: {}", id);
 
-        final Applicant applicant1 = caseData.getApplicant1();
+        if (!YesOrNo.YES.equals(caseData.getSentNotifications()
+            .getSentAwaitingConditionalOrderReminderNotificationSendToApplicant1())) {
```

Comment:
Can these be added as boolean methods on SentNotifications, it will make it easier to read
!YesOrNo.YES.equals(caseData.getSentNotifications()
            .getSentAwaitingConditionalOrderReminderNotificationSendToApplicant1()

something like isAwaitingConditionalOrderReminderNotificationSentToApplicant1